app/templates/projects/show.hbs

```diff
@@ -142,7 +140,7 @@
           {{/if}}
         </div>
         <div class="cell shrink">
-          <span {{action (mut model.needUnderlyingZoning) true }}>
```

Comment:
Why doesn't this need to be updated on the model anymore? Is the idea that since we're done with the wizard at this point, the `need*` booleans are no longer useful and we don't want to use them to control flow so we don't update them to true?

Reply:
After the wizard is completed, the `needs*` booleans no longer matter.